src/applications/vaos/tests/services/appointment/index.v2.unit.spec.js

```diff
@@ -730,4 +733,407 @@ describe('VAOS Appointment service', () => {
       expect(differences).to.be.empty;
     });
   });
+
+  describe('getAppointmentRequests', () => {
+    beforeEach(() => mockFetch());
+    it('should return matching v0 and v2 data for a VA appointment request', async () => {
+      // Given VA appointment request data
+      const data = {
+        id: '1234',
+        start: moment()
+          .add(3, 'days')
+          .format(),
+        email: 'test@va.gov',
+        phone: '2125551212',
+        kind: 'clinic',
+        locationId: '552GA',
+        clinic: '5544',
+        clinicFriendlyName: 'Friendly clinic name',
+        requestedPeriods: [
+          {
+            start: `${moment().format('YYYY-MM-DD')}T00:00:00.000`,
+            end: `${moment().format('YYYY-MM-DD')}T11:59:59.999`,
+          },
+        ],
+        serviceType: 'primaryCare',
+        status: 'proposed',
+        visitType: 'Office Visit',
+      };
+      const startDate = moment()
+        .subtract(30, 'days')
+        .format();
+      const endDate = moment()
+        .add(30, 'days')
+        .format();
+
+      // When we request v2 appointment requests
+      setFetchJSONResponse(
+        global.fetch.withArgs(
+          sinon.match(
+            `/vaos/v2/appointments?start=${startDate}&end=${endDate}&statuses[]=proposed&statuses[]=cancelled`,
+          ),
+        ),
+        {
+          data: [
+            createMockAppointmentByVersion({
+              version: 2,
+              ...data,
+            }),
+          ],
+        },
+      );
+
+      // And When we request v0 appointment requests
+      setFetchJSONResponse(
+        global.fetch.withArgs(
+          sinon.match(
+            `/vaos/v0/appointment_requests?start_date=${startDate}&end_date=${endDate}`,
+          ),
+        ),
+        {
+          data: [
+            createMockAppointmentByVersion({
+              version: 0,
+              ...data,
+            }),
+          ],
+        },
+      );
+
+      const [v0Result, v2Result] = await Promise.all([
+        getAppointmentRequests({
+          startDate,
+          endDate,
+        }),
+        getAppointmentRequests({
+          startDate,
+          endDate,
+          useV2: true,
+        }),
+      ]);
+
+      // These are always different
+      delete v0Result[0].vaos.apiData;
+      delete v2Result[0].vaos.apiData;
+
+      // Then expect these differences
+      // differences format is http://jsonpatch.com/
+      const differences = diff(v2Result[0], v0Result[0]);
+      expect(differences).to.have.deep.members(
+        [
+          { op: 'remove', path: ['description'] },
+          { op: 'remove', path: ['practitioners'] },
+          {
+            op: 'remove',
+            path: ['vaos', 'isPastAppointment'],
```

Comment:
There are some differences in here that look like they're null vs undefined differences. I think it's worth setting the fields explicitly to null in the transforms to trim this list down, if possible. That way the differences list are genuine differences between the data (like the `created`, for example).